bots/issue-scan

```diff
@@ -72,21 +76,34 @@ def main():
 
     kvm = os.access("/dev/kvm", os.R_OK | os.W_OK)
     if not kvm:
-        sys.stderr.write("tests-scan: No /dev/kvm access, not creating images here\n")
+        sys.stderr.write("issue-scan: No /dev/kvm access, not creating images here\n")
 
     try:
         urllib.request.urlopen(REDHAT_PING).read()
         redhat = os.path.exists(os.path.expanduser(REDHAT_CREDS))
     except IOError:
         redhat = False
 
+    # Figure if we're in a Kubernetes namespace. This file will always exist
+    try:
+        with open("/var/run/secrets/kubernetes.io/serviceaccount/namespace", "r") as f:
+            namespace = f.read().strip()
+    except IOError:
+        namespace = None
+
     for result in scan(opts.verbose):
         if not kvm and contains_any(result, KVM_TASKS):
             continue
         elif not redhat and contains_any(result, REDHAT_TASKS):
             continue
         elif contains_any(result, WINDOWS_TASKS):
             continue
+        elif contains_any(result, KUBERNETES_TASKS):
+            if not namespace:
+                continue
+            url = namespace + KUBERNETES_TASKS[0]
+            if url not in result:
+                continue
```

Comment:
It might be useful for debugging to log about this here?

Reply:
What about all the others above? Should they also have log output?

Reply:
Good point, maybe. But I figure we can introduce the logging when that actually becomes a problem.